modules/linear_boltzmann_solvers/lbs_solver/lbs_structs.h

```diff
@@ -17,11 +17,8 @@ namespace lbs
 typedef std::vector<size_t> DirIDs; ///< Direction-IDs
 typedef std::vector<DirIDs> UniqueSOGroupings;
 typedef std::map<size_t, size_t> DirIDToSOMap;
-
-typedef std::vector<double> VecDbl;
-typedef std::vector<VecDbl> MatDbl;
-typedef std::vector<Vector3> VecVec3;
-typedef std::vector<VecVec3> MatVec3;
+// typedef std::vector<Vector3> VecVec3;
+typedef std::vector<std::vector<Vector3>> MatVec3;
```

Comment:
Not technically within scope, but a good opportunity to switch to `using` or to stop using these custom types.

Reply:
I switched these to `using`. I'll have another PR that addresses `typedef` in general.